src/Test.FunctionalTests.Sarif/v2/ConverterTestData/ContrastSecurity/WebGoat.xml.sarif

```diff
@@ -1865,6 +1865,7 @@
         {
           "ruleId": "crypto-bad-mac",
           "ruleIndex": 18,
+          "level": "warning",
```

Comment:
>"level": "warning", [](http://example.com/codeflow?start=10&length=19)

these "level": "warning" lines added is because, in the C# code the way we using the sdk, we explicitly invoke set Level to Warning.
as by design, when user explicitly set Level to Warning we will have this json property generated.
If user does not explicitly invoke set Level, the _level will be null and this json property will not be generated.